ADAL/src/public/mac/ADTokenCache.h

```diff
@@ -46,6 +46,10 @@
     pthread_rwlock_t _lock;
 }
 
+/*! Returns the default cache object using the ADTokenCacheDelegate set in
+    ADAuthenticationSettings */
```

Comment:
-[ADAuthenticationSettings setDefaultStorageDelegate:] will set it on +[ADTokenCache defaultCache]